timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -69,14 +78,22 @@ def run(self):
                     rule.get('es_query'), rule.get('file_name'),
                     tag_list=rule.get('tags'))
                 tags_applied[rule.get('file_name')] += tagged_events_counter
+                if sigma_rule_counter % 10 == 0:
+                    logger.debug('Rule {0:d}/{1:d}'.format(
+                        sigma_rule_counter, len(sigma_rules)))
             except elasticsearch.TransportError as e:
                 logger.error(
                     'Timeout executing search for {0:s}: '
                     '{1!s} waiting for 10 seconds'.format(
                         rule.get('file_name'), e), exc_info=True)
                 # this is caused by to many ES queries in short time range
```

Comment:
I know this comment is old, but can you have it start with a capital and `s/to/too/`

Reply:
also, why have a sleep here? It seems like a duck tape solution to a leak... maybe just fix the underlying issue here, or wait until rules are done before running a new one, can't we do something else to limit the searches (buffer, wait until others are done, etc, etc, etc) rather than have an arbitrary sleep command?